test/harry/main/org/apache/cassandra/harry/model/ASTSingleTableModel.java

```diff
@@ -885,17 +1107,47 @@ public boolean isAllDefined(ImmutableUniqueList<Symbol> selectColumns)
         }
     }
 
-    public ImmutableUniqueList<Symbol> columns(Select select)
+    private ImmutableUniqueList<Symbol> columns(Select select)
```

Comment:
i added this to simplify the tests.  This is the common case for `SELECT col` but will need to be enhanced later on once we cover more things....